ParseAligns/abyss-fixmate.cc

```diff
@@ -327,7 +334,7 @@ static void printHistogramStats(Histogram h)
 		"n: " << h.size() << " "
 		"min: " << h.minimum() << " "
 		"max: " << h.maximum() << " "
-		"ignored: " << n_orig - h.size() << '\n'
+		"ignored as outliers: " << n_orig - h.size() << '\n'
```

Comment:
I'd suggest `trimmed`

Reply:
But this is not really trimming. Trimming in my opinion means removing=altering e.g. an adapter sequece from input reads. Provided it is critical for end-user to understand what is happening with input data (reads/k-mers generally speaking) I think trimming would be unfortunate. I understand you mean "trimming from the distibution" but it is a long shot to understand would there be just 'trimming'. In my opinion `ignored as outliers` is clear to understand, does not require to rename the internal variable `ignored`. Abyss-map just omits those read pairs from further work.

You are the gate-keeper here but ... just my 2c.

Reply:
Trimmed mean is a statistical term. See https://en.wikipedia.org/wiki/Trimmed_estimator
A synonym is truncated mean. https://en.wikipedia.org/wiki/Truncated_mean
I'd be okay with `truncated`.

They're not exactly outliers, which is usually defined as x > mean + n * sigma or x > median + n IQR. That'd probably be a better trimming method than the fixed proportion of trimming that we're doing currently.